HipercowApi/Models/ClusterLoad.cs

```diff
@@ -2,35 +2,18 @@
 
 namespace HipercowApi.Models
 {
+    using System.Diagnostics.CodeAnalysis;
+
     /// <summary>
     /// A class containing information about a cluster's current load.
     /// </summary>
-    public class ClusterLoad
-    {
-        /// <summary>
-        /// Initializes a new instance of the <see cref="ClusterLoad"/> class.
-        /// </summary>
-        /// <param name="name">The name of the requested cluster.</param>
-        /// <param name="nodeLoads">
-        /// A list of NodeLoad objects, giving the current number of busy cores,
-        /// total number of cores, and state (online or offline) of each node.
-        /// </param>
-        public ClusterLoad(
-            string name,
-            List<NodeLoad> nodeLoads)
-        {
-            this.Name = name;
-            this.NodeLoads = nodeLoads;
-        }
-
-        /// <summary>
-        /// Gets or sets the name of the cluster.
-        /// </summary>
-        public string Name { get; set; }
-
-        /// <summary>
-        /// Gets or sets the list of NodeLoad objects.
-        /// </summary>
-        public List<NodeLoad> NodeLoads { get; set; }
-    }
+    /// <param name="Name">The name of the requested cluster.</param>
+    /// <param name="NodeLoads">
+    /// A list of NodeLoad objects, giving the current number of busy cores,
+    /// total number of cores, and state (online or offline) of each node.
+    /// </param>
+    [ExcludeFromCodeCoverage]
```

Comment:
I'm also surprised this is not showing up as being used in code coverage if these are being created by the tested code, might be worth digging into that a little. 

Since it looks like you're going to have this issue with every model type, it might be worth excluding the whole Models folder from code coverage. But it seems a shame to have to do that, since if it was working code coverage could alert to having unused models still in the codebase for example.

Reply:
Yeah, this felt like a code-cov problem rather than a coding problem, since I get a different result depending on how the models are expressed. If I do it kind of verbatim, I think it's ok, if I let it refactor into something more concise, or use a record, then I get mixed results...